pipelines/pom.xml

```diff
@@ -304,7 +304,20 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
         <configuration>
-          <release>11</release>
+          <source>17</source>
+          <target>17</target>
+          <compilerArgs>
+            <!-- Enable access to the packages at compile time -->
+            <arg>--add-exports=java.base/jdk.internal.misc=ALL-UNNAMED</arg>
```

Comment:
This change was needed at both the places i.e. at `pipelines/pom.xml` so that the packages are available during compile and run times (test cases), as well as `pipelines/controller/pom.xml` so that these options gets added as manifest entries in the `bundled` fat jar.